Explorer/Assets/Scripts/CrdtEcsBridge/ComponentWriter/ECSToCRDTWriter.cs

```diff
@@ -0,0 +1,70 @@
+using CRDT;
+using CRDT.Protocol;
+using CRDT.Protocol.Factory;
+using CrdtEcsBridge.OutgoingMessages;
+using CrdtEcsBridge.Serialization;
+using System;
+using System.Collections.Generic;
+
+namespace CrdtEcsBridge.ECSToCRDTWriter
+{
+    public class ECSToCRDTWriter : IECSToCRDTWriter
+    {
+        private readonly ICRDTProtocol crdtProtocol;
+        private readonly IOutgoingCRTDMessagesProvider outgoingCRDTMessageProvider;
+        private readonly Dictionary<int, IComponentSerializer> serializers;
```

Comment:
You don't need it: you can use `ISDKComponentsRegistry`, `SDKCOmponentBridge` already contains information about `Serializer`